products/libllbuild/include/llbuild/buildsystem.h

```diff
@@ -111,6 +111,7 @@ typedef enum LLBUILD_ENUM_ATTRIBUTES {
   llb_buildsystem_command_result_failed = 1,
   llb_buildsystem_command_result_cancelled = 2,
   llb_buildsystem_command_result_skipped = 3,
+  llb_buildsystem_command_result_fatal = 4,
```

Comment:
Can you please add a doc comment to this member explaining what it's used for? I know the others don't have them right now, but given that we have 5 values now, it would be good to start :)

Reply:
Yep, makes sense. Especially since that's the one with a more special behavior